src/drivers/amd/renoir/acp_bt_dai.c

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+//Copyright(c) 2021 AMD. All rights reserved.
+//
+//Author:	Basavaraj Hiregoudar <basavaraj.hiregoudar@amd.com>
+//		Anup Kulkarni<anup.kulkarni@amd.com>
+//		Bala Kishore <balakishore.pati@amd.com>
+
+#include <sof/audio/component.h>
+#include <sof/drivers/acp_dai_dma.h>
+#include <sof/drivers/interrupt.h>
+#include <sof/lib/alloc.h>
+#include <sof/lib/dai.h>
+#include <sof/lib/dma.h>
+#include <sof/lib/uuid.h>
+#include <ipc/dai.h>
+#include <ipc/topology.h>
+#include <platform/fw_scratch_mem.h>
+#include <sof/lib/io.h>
+#include <platform/chip_offset_byte.h>
+
+/* 4abd71ba-8619-458a-b33f-160fc0cf809b */
+DECLARE_SOF_UUID("btdai", btdai_uuid, 0x4abd71ba, 0x8619, 0x458a,
+		0xb3, 0x3f, 0x16, 0x0f, 0xc0, 0xcf, 0x80, 0x9b);
+
+DECLARE_TR_CTX(btdai_tr, SOF_UUID(btdai_uuid), LOG_LEVEL_INFO);
+
+static int btdai_context_store(struct dai *dai)
+{
+	//TODO
+	return 0;
+}
+
+static int btdai_context_restore(struct dai *dai)
+{
+	//TODO
+	return 0;
+}
```

Comment:
Not related to this PR, but I think we can probable remove the context save and restore from the general codebase as I don't think they will ever be used. Host drivers and userspace take core of audio context. I will do this work for v1.9

Reply:
Addressed the Review Comment. removed the API's .Done